ts/util/table-builder.ts

```diff
@@ -331,9 +333,18 @@ export class TableBuilder<ParentType, ChildType> {
       reload();
     };
     return new BasicDropdownOption(filter.title, () => {
-      const filterTextInput = new TextInput(filter.title, onClose);
+      if (!filter.url) {
+        throw new Error(
+          `Text input filter ${filter.title} has no autocomplete rest URL`
+        );
+      }
+      const filterTextInput = new TextInput(filter.title, onClose, filter.url);
+      filterContainer.insertBefore(
+        filterTextInput.getContainerTag(),
+        filterContainer.lastChild
+      );
       filterContainer.insertBefore(
-        filterTextInput.getTag(),
+        filterTextInput.getDatalistTag(),
```

Comment:
The datalist should be inside the container, so it doesn't need to be added/removed separately